### DISCUSSION TITLE
Stronghold Recipes

### DISCUSSION THREAD

**Discussion by nothingismagick** [iotaledger/stronghold.rs#102] [Ideas]
This discussion is about ideas, pseudocode and things that engineers can do with stronghold.

**Comment by logiota** [comment]
Create a Trezor fork with Stronghold that would run on a Pi Zero and get something better than a secure chip for the main system.
<img width="745" alt="Screenshot 2021-01-27 at 00 55 26" src="https://user-images.githubusercontent.com/45556023/105922159-5e378a80-603a-11eb-92c5-dd748f0ad355.png">

**Reply by nothingismagick** [reply]
Do you mean specifically https://github.com/trezor/trezor-suite @logiota ? Or are you talking about something else that does not require a V8 engine?

**Reply by logiota** [reply]
I'm talking about this one:
https://github.com/trezor/trezor-firmware

**Reply by logiota** [reply]
How you use the usb communication on the main system doesn't matter so much, the important thing is not being able to extract the private keys from the hardware wallet.
https://wallet.trezor.io/#/bridge should not be too hard to use as a interface or even relying on a direct usb connection.

**Comment by logiota** [comment]
I guess the first thing to do is check if there is some way to use the current blockchain based signatures scheme to assist in signing iota transactions.

**Reply by nothingismagick** [reply]
I still don't really understand what you mean. Stronghold is able to manufacture keys with slip10 and sign transactions already (for IOTA).
